.ci/magician/cmd/test_terraform_vcr.go

```diff
@@ -393,15 +403,22 @@ func runReplaying(runFullVCR bool, services map[string]struct{}, vt *vcr.Tester)
 	var testDirs []string
 	var replayingErr error
 	if runFullVCR {
-		fmt.Println("runReplaying: full VCR tests")
-		result, replayingErr = vt.Run(vcr.Replaying, provider.Beta, nil)
+		fmt.Println("run full VCR tests")
```

Comment:
nit: I think this got lost in the merge conflict resolution

```suggestion
		fmt.Println("runReplaying: full VCR tests")
```